katas/content/nonlocal_games/index.md

```diff
@@ -53,9 +51,147 @@ To start with, let's take a look at how you would play the classical variant of
+$$\begin{bmatrix}
+    0.7071 & 0 & 0.7071 & 0\\
+    0 & 0.7071 & 0 & 0.7071\\
+    0.7071 & 0 & -0.7071 & 0\\
+    0 & 0.7071 & 0 & -0.7071
+\end{bmatrix}$$
+to the Bell state, resulting in a vector of:
+$$\begin{bmatrix}
+    0.5\\
+    0.5\\
+    0.5\\
+    -0.5
+\end{bmatrix}$$
+
+Now, one of the two rotation operators is applied depending on what bit Bob holds, transforming this vector into:
+$$\begin{bmatrix}
+    0.6533 \\
+    0.2706 \\
+    0.2706 \\
+    -0.6533
+\end{bmatrix} \text{ or }
+\begin{bmatrix}
+    0.2706\\
+    0.6533\\
+    0.6533\\
+    -0.2706
+    \end{bmatrix}$$
+
+When Bob holds 0, they still want to return the same parity, which they again do with 85.36% probability (left-hand vector above).
+But when Bob holds 1, the AND condition is now true and the players want to answer in opposite parity. This corresponds to the second and third elements of the right-hand vector above.
+Thanks to the "magic" of the combination of the counterclockwise rotation and Hadamard transform, they now do this with probability $(0.6533)^2 + (0.6533)^2 = 0.8536$ and thus 85.36% becomes their win odds once more.
+
+## Side notes
+
+- If Bob never rotated his qubit, their entangled state would remain the Bell state if Alice held bit 0 and the state corresponding to $\frac12 \big(|00\rangle + |01\rangle + |10\rangle - |11\rangle\big)$ if Alice held bit 1.
+
+While she and Bob would have a 100% success probability against Alice's 0 bit, they would have only a 50% chance of success if she held bit 1, and thus their win chance would revert to the 75% of the classical strategy again.
+- It can be proven that Alice and Bob cannot surpass an overall win probability of 85.36% in the CHSH game. This entails a higher-level discussion of quantum observable theory, for instance see [Tsirelson's bound](https://en.wikipedia.org/wiki/Tsirelson's_bound).
+
+@[section]({
+    "id": "nonlocal_games__chsh_e2e",
+    "title": "Running CHSH Game End to End"
+})
+
+Putting together the building blocks we've implemented into a strategy is very simple:
+- Allocate two qubits and prepare a Bell state on them.
+- Send one of the qubits to Alice and another to Bob (this step is "virtual", not directly reflected in Q# code, other than making sure that Alice and Bob each act on their qubit only).
+- Have Alice and Bob perform their measurements on their respective qubits using `AliceQuantum` and `BobQuantum` operations.
+- Release(reset) used qubits
+- Return their measurement results.
+
+In the example below you can compare classical and quantum results: first boolean value indicates win for classical Alice and Bob, second boolean is win for quantum Alice and Bob.
+You may play with the code and check if there is a difference in results when
+1. referee picks non-random bits
+2. Bob's qubit is measured first
+3. Alice and Bob get not entangled qubit pair
```

Comment:
Nice examples!! They'll need to be a separate paragraph for better text flow
```suggestion
You may play with the code and check if there is a difference in results when
1. The referee picks non-random bits. How can the referee minimize Alice and Bob's win probability?
2. Bob's qubit is measured first.
3. Alice and Bob get unentangled qubit pair.
```

Reply:
Suggestions were indented ('>' is used).
Screenshot of result is attached.
![index_suggestion_indentation](https://github.com/user-attachments/assets/0a901b00-1d4b-4335-b609-aedd2e00f8e0)